src/Angular.js

```diff
@@ -97,6 +97,9 @@
   NODE_TYPE_DOCUMENT,
   NODE_TYPE_DOCUMENT_FRAGMENT
 */
+/* global
+  console
```

Comment:
You can avoid this by accessing `console` off `window` (not sure what is better).